test/test.js

```diff
@@ -1,10 +1,10 @@
 import { expect } from 'chai';
 import { assert } from 'chai';
 import Debug from 'debug';
-const debug = new Debug('test:generatePassword:');
-import _ from 'lodash';
```

Comment:
In case you're wondering why I moved the const declaration down: `import` statements are required to be above everything but babel does not follow the spec exactly so this never caused an error.